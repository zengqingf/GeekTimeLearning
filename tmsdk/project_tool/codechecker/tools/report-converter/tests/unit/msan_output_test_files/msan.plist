<?xml version="1.0" encoding="UTF-8"?>
<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
<plist version="1.0">
<dict>
	<key>diagnostics</key>
	<array>
		<dict>
			<key>category</key>
			<string>unknown</string>
			<key>check_name</key>
			<string>MemorySanitizer</string>
			<key>description</key>
			<string>use-of-uninitialized-value</string>
			<key>issue_hash_content_of_line_in_context</key>
			<string>bbcfe23432808e76f0c771aacb4c377a</string>
			<key>location</key>
			<dict>
				<key>col</key>
				<integer>7</integer>
				<key>file</key>
				<integer>0</integer>
				<key>line</key>
				<integer>7</integer>
			</dict>
			<key>notes</key>
			<array>
				<dict>
					<key>depth</key>
					<integer>0</integer>
					<key>kind</key>
					<string>note</string>
					<key>location</key>
					<dict>
						<key>col</key>
						<integer>7</integer>
						<key>file</key>
						<integer>0</integer>
						<key>line</key>
						<integer>7</integer>
					</dict>
					<key>message</key>
					<string>    #0 0x4940da in main files/msan.cpp:7:7
    #1 0x7fed9df58b96 in __libc_start_main (??)
    #2 0x41b2d9 in _start (??)
</string>
				</dict>
			</array>
			<key>path</key>
			<array>
				<dict>
					<key>depth</key>
					<integer>0</integer>
					<key>kind</key>
					<string>event</string>
					<key>location</key>
					<dict>
						<key>col</key>
						<integer>7</integer>
						<key>file</key>
						<integer>0</integer>
						<key>line</key>
						<integer>7</integer>
					</dict>
					<key>message</key>
					<string>    #0 0x4940da in main files/msan.cpp:7:7</string>
				</dict>
				<dict>
					<key>depth</key>
					<integer>0</integer>
					<key>kind</key>
					<string>event</string>
					<key>location</key>
					<dict>
						<key>col</key>
						<integer>7</integer>
						<key>file</key>
						<integer>0</integer>
						<key>line</key>
						<integer>7</integer>
					</dict>
					<key>message</key>
					<string>use-of-uninitialized-value</string>
				</dict>
			</array>
			<key>type</key>
			<string>msan</string>
		</dict>
	</array>
	<key>files</key>
	<array>
		<string>files/msan.cpp</string>
	</array>
	<key>metadata</key>
	<dict>
		<key>analyzer</key>
		<dict>
			<key>name</key>
			<string>msan</string>
		</dict>
		<key>generated_by</key>
		<dict>
			<key>name</key>
			<string>report-converter</string>
			<key>version</key>
			<string>x.y.z</string>
		</dict>
	</dict>
</dict>
</plist>
