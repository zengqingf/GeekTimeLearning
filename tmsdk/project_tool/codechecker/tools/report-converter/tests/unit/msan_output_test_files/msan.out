==12680==WARNING: MemorySanitizer: use-of-uninitialized-value
    #0 0x4940da in main files/msan.cpp:7:7
    #1 0x7fed9df58b96 in __libc_start_main (??)
    #2 0x41b2d9 in _start (??)

SUMMARY: MemorySanitizer: use-of-uninitialized-value files/msan.cpp:7:7 in main
Exiting